src/Yoti.Auth/DocScan/DocScanService.cs

```diff
@@ -82,7 +80,7 @@ public async Task<GetSessionResult> GetSession(string sdkId, AsymmetricCipherKey
                 .WithHttpMethod(HttpMethod.Get)
                 .WithBaseUri(ApiUri)
                 .WithEndpoint(sessionEndpoint)
-                .WithQueryParam("sdkId", sdkId)
+                .WithQueryParam("sdkId", sdkId)  
```

Comment:
I think a bit of whitespace has crept in here. 
```suggestion
                .WithQueryParam("sdkId", sdkId)
```
Ideally we'd have a pre-commit step to trim things like this. We can add this at a later stage. 
Something like [this ](https://github.com/dotnet/format/blob/main/docs/integrations.md#pre-commitcom-hook-to-reformat)might work

Reply:
Yes I saw that once PR was up on github. It's annoying as VS doesn't (with my settings anyhow - almost certainly configurable) pick up on these, but GitHub's diff viewer does. VS settings make sense for C# in VS (as that whitespace is obviously irrelevant to the MSIL and/or runtime code), GitHub needs to deal with Python etc so does it's diffs differently. Will see if I can amend my VS settings to match GitHub so we avoid going forwards. Thanks

Reply:
Will amend with new commit. Thanks